src/sentry/relay/globalconfig.py

```diff
@@ -0,0 +1,5 @@
+def get_global_config():
+    """Return the global configuration for Relay."""
+    # TODO(iker): Add entries for the global config as needed, empty during
+    # development.
+    return {}
```

Comment:
So, currently we do not return anything at all if the global config is requested?

Maybe it would be great to send at least 1 key, so it we can write some e2e tests to check the proper behaviour

Reply:
That's a valid approach, even if it looks strange to emit an empty config. To add new keys here, there are more changes needed, such as removing them from the project configs and bumping extraction version.